app/src/index.tsx

```diff
@@ -1,7 +1,10 @@
 import React from "react";
+// Rest of your app code
 import ReactDom from "react-dom/client";
 
-import { App } from "./App";
+import "vite/modulepreload-polyfill";
```

Comment:
Can you link why this is needed? https://vitejs.dev/config/build-options#build-modulepreload